lib/models/theme.dart

```diff
@@ -32,6 +32,59 @@ class AppBrightnessType {
   ];
 }
 
+class AppStartTabGh {
+  static const news = 0;
+  static const notification = 1;
+  static const trending = 2;
+  static const search = 3;
+  static const me = 4;
+  static const explore = 5;
+  static const groups = 6;
+  static const teams = 7;
+
+  static const values = [
+    AppStartTabGh.news,
+    AppStartTabGh.notification,
+    AppStartTabGh.trending,
+    AppStartTabGh.search,
+    AppStartTabGh.me,
+  ];
+}
```

Comment:
Since we do not change the order of tabs, it seems these enums are not necessary anymore?

We could just save the index.